test/mlir/test/gc/gpu-runner/lit.local.cfg

```diff
@@ -1,2 +1,4 @@
 if not config.gc_use_imex:
-    config.unsupported = True
\ No newline at end of file
+    config.unsupported = True
+else: # FIXME: Remove this when the GPU runner is implemented.
+    config.unsupported = True
```

Comment:
This is the PR that implements GPU runner, right?

Reply:
No, just runtime, but not runner.